conf/hasta.config

```diff
@@ -21,6 +22,10 @@ params {
 process {
     executor = 'slurm'
     clusterOptions = { "-A $params.priority ${params.clusterOptions ?: ''}" }    
+    
+    withName: QUALIMAP_BAMQC {
+        ext.args = { "--java-mem-size=${task.memory.giga / 1.15 as long}G" }
+    }
```

Comment:
We should put this part in the hasta raredisease config
https://github.com/nf-core/configs/blob/123706519e2016adb64531181c0e4c465b028af1/conf/pipeline/raredisease/hasta.config#L3

Reply:
Fixed!